src/main/src/jetty/geoserver_data/workspaces/imos/geoserver/srs_timeseries_url/gogoduck.xml

```diff
@@ -0,0 +1,3 @@
+<gogoduck>
+    <ncks.parameters>-v;time,lat,lon,dt_analysis,l2p_flags,quality_level,satellite_zenith_angle,sea_surface_temperature,sses_bias,sses_count,sses_standard_deviation,sst_dtime</ncks.parameters>
```

Comment:
I thought we were going to put this at the data store level - thats were it differs for all our datasets.